snmp/datadog_checks/snmp/data/default_profiles/avtech-roomalert-3s.yaml

```
@@ -0,0 +1,83 @@
+extends:
+  - _base.yaml
+  - avtech.yaml
```

the `avtech.yaml` profile already extends `_base.yaml`
```suggestion
extends:
  - avtech.yaml
```